pom.xml

```diff
@@ -214,7 +214,7 @@
 		<hibernate.dialect>org.hibernate.dialect.PostgreSQLDialect</hibernate.dialect>
 		<hibernate.connection.driver_class>org.postgresql.Driver</hibernate.connection.driver_class>
 		<hibernate.hbm2ddl.auto>update</hibernate.hbm2ddl.auto>
-		<hibernate.show_sql>true</hibernate.show_sql>
+		<hibernate.show_sql>false</hibernate.show_sql>
```

Comment:
My two cents about this: instead of modifying the common pom.xml file, you should rather maintain your own Maven settings.xml file as very recently documented here: http://wiki.sigmah.org/doku.php?id=contributorguide:preparebuildenvironment#step_3your_first_build 

Enjoy !